src/api/schema.ts

```diff
@@ -38,6 +40,8 @@ export const lists = {
       resolveInput: async ({ operation, resolvedData, context }) => {
         if (operation === 'create') {
           try {
+            // const emailResponse = await notify.sendEmail(EMAIL_TEMPLATE_IDS.ACCOUNT.CONFIRM_EMAIL, 'Tony.Barnes@ukexportfinance.gov.uk');
```

Comment:
can this be removed or TODO added for removal/change

Reply:
Removed sorry, typo during a POC.